static/scss/style.scss

```diff
@@ -1119,7 +1148,7 @@ img.cand-pic {
   font-size: 1.1em;
   line-height: 150%;
   position: relative;
-  min-height: 320px;
+  min-height: 100px;
```

Comment:
Does this need to have a min-height to look right? If so, maybe it should be measured in `em`, like 2em is approximately two lines?